cli/azd/pkg/input/console.go

```diff
@@ -32,6 +32,8 @@ type AskerConsole struct {
 	interactive bool
 	asker       Asker
 	handles     ConsoleHandles
+	// the writer the console was constructed with, and what we reset to when SetWriter(nil) is called.
+	defaultWriter io.Writer
 	// the current writer, may differ from handles.Stdout when SetWriter has been
 	// called.
```

Comment:
I guess this comment might be outdated given we also allow construction with a different writer...

```suggestion
	// the writer which output is written to. 
```